apps/voting/app/src/components/VoteCard/VoteCard.js

```diff
@@ -56,6 +81,14 @@ function VoteCard({ vote, onOpen }) {
   // “highlighted” means either focused or hovered
   const [highlighted, setHighlighted] = useState(false)
 
+  const disputableStatus =
+    vote.disputable && DISPUTABLE_VOTE_STATUSES.get(vote.disputable.status)
```

Comment:
Something to consider when this goes into the disputable app – if the disputable property is always available and with a status, we could remove the guards.